### PR TITLE
Clarified OS Light/Dark Mode functionality

### PR DESCRIPTION
**Pull request by cwarnermm**
Docs listening: "Dark Theme (Desktop app only)... applies Light or Dark Mode based on the System ...' Where is this "Dark Theme"? Looks like that feature is missing in 5.1.1.."
- updated the Dark Theme documentation to clarify how the OS system preference is applied to the Desktop App with an animated GIF

### CODE REVIEW

**Comment by cwarnermm** [comment]
@hmhealey - Can you confirm whether this functionality is supported across all of our supported platforms, including Windows and Linux?

**Comment by hmhealey** [comment]
I'm not sure where this is supported. Re-assigning to @devinbinnie since he knows more about the desktop app

File: source/welcome/customize-your-theme.rst [hunk 1]
```diff
@@ -269,4 +269,7 @@ Night Owl Dark Theme
 Dark Theme (Desktop app only)
 -----------------------------
 
-The Mattermost Desktop App applies Light or Dark Mode based on the System Preferences appearance setting of your computer's operating system to your Mattermost instance.
+Mattermost applies the Light Mode or Dark Mode system preference configured on your computer to the Mattermost Desktop App.
```
Comment (justinegeffen) [inline source/welcome/customize-your-theme.rst:272]:
```suggestion
The system display preference that you set on your computer (e.g., Light Mode or Dark Mode) is also applied to the Mattermost desktop app.
```

File: source/welcome/customize-your-theme.rst [hunk 2]
```diff
@@ -269,4 +269,7 @@ Night Owl Dark Theme
 Dark Theme (Desktop app only)
```
Comment (justinegeffen) [inline source/welcome/customize-your-theme.rst:269]:
```suggestion
Dark Theme (desktop app only)
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Added rewording suggestion and approved for expediency.

File: source/welcome/customize-your-theme.rst [hunk 3]
```diff
@@ -252,7 +252,10 @@ Night Owl Dark Theme
...
 
-Dark Theme (Desktop app only)
+Dark Theme (desktop app only)
 -----------------------------
 
-The Mattermost desktop app applies Light or Dark Mode based on the System Preferences appearance setting of your computer's operating system to your Mattermost instance.
+The system display preference that you set on your computer (e.g., Light Mode or Dark Mode) is also applied to the Mattermost desktop app.
+
+.. image:: ../images/dark-theme-via-os.gif
+  :alt: The system preference you set for Light Mode or Dark Mode on your computer is automatically applied to the Mattermost Desktop App.
```
Comment (devinbinnie) [inline source/welcome/customize-your-theme.rst:261]: For Linux, this is done manually through the View menu.

**Review by devinbinnie — APPROVED** [review]

**Pull request merged**